ocpp_codec/structure.py

```diff
@@ -45,6 +45,9 @@ class Call(OCPPMessage):
     action: str
     payload: dict
 
+    def __post_init__(self):
+        self.messageTypeId = MessageTypeEnum.CALL
```

Comment:
Since the field was defined as `init=False`, you can simply assign it  at the class level:

```python
class Call:
    messageTypeId = MessageTypeEnum.CALL
```